geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GatewayReceiverCreateFunction.java

```diff
@@ -154,6 +165,10 @@ private Object newInstance(String className)
     return ClassPathLoader.getLatest().forName(className).newInstance();
   }
 
+  private boolean doesSomeGatewayReceiverAlreadyExists(Cache cache) {
```

Comment:
Only one gateway-receiver is allowed per server, so the method name shouldn't suggest plurality. I suggest `gatewayReceiverExists`

Reply:
I'm bad at names.  Yours is better, thanks.